static/calendar.ics

```diff
@@ -26,8 +26,15 @@ END:VTIMEZONE
 {% for termin in site.data.calendar -%}
 {% assign start_uhrzeit = termin.uhrzeit | remove: "h" | remove: "." | append: "00" -%}
 {% assign start = termin.datum | remove: "-" | append: "T" | append: start_uhrzeit -%}
-{% assign end_uhrzeit = termin.uhrzeit | remove: "h" | remove: "." | plus: 200 | append: "00" -%}
-{% assign end = termin.datum | remove: "-" | append: "T" | append: end_uhrzeit -%}
+{% if termin.dauer -%}
+{%  assign dauer = termin.dauer | times: 100 -%}
+{% else -%}
+{%  assign dauer = 100 -%}
+{% endif -%}
+{% assign end_uhrzeit = termin.uhrzeit | remove: "h" | remove: "." | plus: dauer -%}
+{% assign dauer_tage = end_uhrzeit | divided_by: 2400 -%}
+{% assign end_uhrzeit = end_uhrzeit | modulo: 2400 -%}
+{% assign end = termin.datum | remove: "-" | plus: dauer_tage | append: "T" | append: end_uhrzeit | append: "00" -%}
```

Comment:
Pfusch!

Aber wird schon tun, außer wenn Termine sich über ein Monatsende hinweg erstrecken. Das kann jemand in der Zukunft lösen...